spec/brainstem-collection-spec.coffee

```diff
@@ -304,6 +303,17 @@ describe 'Brainstem.Collection', ->
         options = { page: 1, perPage: 5 }
         collection = new App.Collections.Posts(null, options)
 
+      it 'returns a promise with jqXhr methods', ->
+        respondWith(server, '/api/posts?per_page=5&page=1', resultsFrom: 'posts', data: { posts: posts1 })
+
+        jqXhr = $.ajax()
+        promise = collection.fetch()
+
+        for key, value of jqXhr
+          object = {}
+          object[key] = jasmine.any(value.constructor)
```

Comment:
I see what you did here -- naice! This should probably be the standard practice so it doesn't serialize a huge object that makes it impossibly hard to figure out which key-value is missing.

![he-he-he](http://i.imgur.com/meftV0q.gif?noredirect)